doc/install_macos.txt

```diff
@@ -0,0 +1,197 @@
+.. include:: css.txt
+
+.. _install_macos:
+
+
+Mac OS Installation Instructions
+################################
+
+There are various ways to install Theano dependencies on a Mac. Here
+we describe the process in detail with Canopy, Anaconda, Homebrew or
+MacPorts but if you did it differently and it worked, please let us
+know the details on the `theano-users`_ mailing-list, so that we can
+add alternate instructions here.
+
+.. _theano-users: http://groups.google.com/group/theano-users?pli=1
+
+In academia: Enthought Canopy
```

Comment:
Agreed. I am more inclined not to have a reference to Conda at all. I think it goes beyond the scope of the installation instructions (i.e. we should provide *a* way to install Theano, not every possible way to install a python package). 

Nonetheless, I remember you suggested to have a more expanded requirement section with a reference to conda. I don't remember your proposal exactly (hopefully you wrote it somewhere here) but I remember agreeing with you. In case you didn't forget, can you refresh my memory?